Assets/LeapMotion/Scripts/Utils/Hands.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.SceneManagement;
+
+namespace Leap.Unity {
+
+  public static class Hands {
+
+    private static LeapServiceProvider s_provider;
+    private static GameObject s_leapRig;
+
+    static Hands() {
+      InitStatic();
+      SceneManager.activeSceneChanged += InitStaticOnNewScene;
+    }
+
+    private static void InitStaticOnNewScene(Scene unused, Scene unused2) {
+      InitStatic();
+    }
+
+    private static void InitStatic() {
+      s_provider = GameObject.FindObjectOfType<LeapServiceProvider>();
+      s_leapRig = s_provider.transform.parent.parent.parent.gameObject;
+    }
+
+    /// <summary>
+    /// Static convenience accessor for the Leap Rig GameObject.
+    /// </summary>
+    public static GameObject Rig {
```

Comment:
It's the GameObject with Rig in the name -- the camera's parent. This is the thing you would move if you wanted to translate the player's position in space. I agree, it's really dangerous and we shouldn't assume it's going to be in a given place down the hierarchy, but we should have a static accessor for the VR Camera's parent. How can we reliably get it? Camera.main.transform.parent?